study-builder/studies/mbc/snippets/common/i18n/es.conf

```diff
@@ -0,0 +1,524 @@
+    # ddp-study-server/study-builder/studies/snippets/bool-question-yes-no-required.conf: 15
+"bool-question-yes-no-required_validations_01_hintTemplate_01"="[ES]Please choose yes or no"
+    # ddp-study-server/study-builder/studies/snippets/date-question-dob-required.conf: 26
+"date-question-dob-required_validations_01_hintTemplate_01"="[ES]Please enter your date of birth in MM DD YYYY format"
+    # ddp-study-server/study-builder/studies/snippets/date-question-dob-required.conf: 47
+"date-question-dob-required_validations_02_hintTemplate_01"="[ES]Please enter your date of birth in MM DD YYYY format"
+    # ddp-study-server/study-builder/studies/snippets/picklist-question-country-required.conf: 19
+"picklist-question-country-required_promptTemplate_01"="[ES]What country do you live in?"
+    # ddp-study-server/study-builder/studies/snippets/picklist-question-country-required.conf: 40
+"picklist-question-country-required_validations_01_hintTemplate_01"="[ES]Country is required"
+    # ddp-study-server/study-builder/studies/snippets/picklist-question-country-required.conf: 58
+"picklist-question-country-required_picklistLabelTemplate_01"="[ES]Choose country..."
+    # ddp-study-server/study-builder/studies/snippets/picklist-question-country-required.conf: 78
+"picklist-question-country-required_picklistOptions_01_optionLabelTemplate_01"="[ES]United States"
+    # ddp-study-server/study-builder/studies/snippets/picklist-question-country-required.conf: 100
+"picklist-question-country-required_picklistOptions_02_optionLabelTemplate_01"="[ES]Canada"
```

Comment:
... do we actually need to translate the country names?

Reply:
Well:
United States -> Estados Unidos
Egypt -> Egipto
Cyprus -> Chipre
England -> Inglaterra

I think anyone who speaks Spanish will recognize the English version of the names, but they are different.

Reply:
Won't that lead to an accessibility issue for users trying to select a language if the list is not in their native language, or is a user mistakenly chooses an incorrect language? Best practices here are to use the local format and and spelling of the language.